src/knock.ts

```diff
@@ -59,6 +73,28 @@ class Knock {
     return this.workflows.trigger(workflowKey, properties);
   }
 
+  signToken(userId: string, options?: SignTokenOptions) {
+    if (!this.signingKey) throw new NoSigningKeyProvidedException();
+
+  // JWT NumericDates specified in seconds:
+  const currentTime = Math.floor(Date.now() / 1000);
+  
+  // Default to 1 hour from now
+  const expireInSeconds = options?.expiresInSeconds ?? 60 * 60;
+
+  return jwt.sign(
+      {
+        sub: userId,
+        iat: currentTime,
+        exp: currentTime + expireInSeconds,
+      },
+      this.signingKey,
```

Comment:
We should have them pass this into the function rather than in the constructor IMO or do the sensible default that we're doing by reading from the env var